### DISCUSSION TITLE
Problem Debugging AutoCAD Plugin with Visual Studio: Assembly Loaded from Temp Folder

### DISCUSSION THREAD

**Discussion by rodriggo-afk** [chuongmep/CadAddinManager#11] [General]
Hello,

I'm facing a challenge developing a plugin for AutoCAD and would like to ask for your help or insights.

I recently started working on a plugin for AutoCAD using Visual Studio. The project compiles without problems, and the .pdb file for debugging is generated correctly in the C:\Users\User\source\repos\CivilPset\CivilPset\bin\x64\Debug folder. However, when I try to debug the plugin in AutoCAD using the AddIn Manager, I am encountering a peculiar situation: Visual Studio is not loading the assembly from the Debug folder as expected. Instead, the plugin is loaded from a temporary folder (C:\Users\User\AppData\Local\Temp\AcadAddins\...), and this has prevented breakpoints from being hit during debugging. The error message displayed by Visual Studio is "Binary was not built with debug information."

Here are some points I checked:

The plugin is compiled in Debug mode, and the .pdb file is present in the project's output folder.
The project settings in Visual Studio appear to be correct, including the options for generating debug information.
I tried manually copying the DLL and PDB file to the mentioned temporary folder, but the problem persists.
This behavior is puzzling, especially because Visual Studio and AddIn Manager appear to be redirecting execution to a temporary folder rather than using the project output folder where the debug files reside. This results in the inability to reach breakpoints and effectively debug the code.

I would like to know if anyone has faced a similar situation or has suggestions on how to resolve this issue. Is there a specific setting in Visual Studio, AutoCAD, or the plugin loading process that I may have missed? Any help or direction would be greatly appreciated.

Thank you in advance for your assistance and time.

**Comment by chuongmep** [comment]
Hi @rodriggo-afk , the latest version can resolve your issue right now .
